src/index.css

```diff
@@ -91,25 +112,36 @@ body {
   font-family: Poppins, sans-serif;
   font-weight: bold;
   font-size: 30px;
+  color: white;
   margin-right: 20px;
 }
 
 .reload {
   width: 40px;
   height: 40px;
+  background-color: aqua;
   margin-left: 20px;
   border-radius: 50px;
 }
 
+.reload:hover {
+  background-color: #fff;
+  color: rgb(232, 103, 123);
+  border: 3px solid rgb(232, 103, 123);
+}
+
 .context {
+  border: 4px solid rgb(255, 255, 255);
   width: 330px;
   overflow: scroll;
-  height: 55vh;
+  height: 50vh;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding-top: 40px;
+  margin-bottom: 20px;
+  background-color: rgb(241, 172, 178);
+  padding-top: 20px;
 }
```

Comment:
- [ ] I would suggest you please implement the changes requested by the previous reviewer. When I add the name and score then the first name and score are hidden. please fix it
### Your design
![ssss](https://user-images.githubusercontent.com/81727988/189332627-2458a642-fb6b-4b4d-bbf4-49b52571d7b6.png)